packages/wonder-blocks-switch/src/themes/default.ts

```diff
@@ -0,0 +1,66 @@
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+const theme = {
+    color: {
+        bg: {
+            primary: {
```

Comment:
**suggestion:** I think the `primary/secondary/etc` category doesn't apply here.

For context, in `Button`, I used these terms b/c it was descriptive for the `kind` prop which is used by the same component (`<button/>`).

As I understand (please correct me if I'm wrong), in switch, these variants are used for different elements/surfaces within the component (`switch`, `slider`, `icon`).

My recommendation is that if we go with a more semantic approach (from the component-level perspective) is to provide names that align more to the structure of the component:

e.g.
- primary => `color.bg.switch`.
- secondary => `color.bg.slider`
- icon => `color.bg.icon`

You can read more about that concept here:
https://spectrum.adobe.com/page/design-tokens/#Use-component-specific-tokens-for-their-respective-component
https://uxdesign.cc/design-tokens-cheatsheet-927fc1404099#:~:text=Anatomy%20of%20a%20Design%20Token%20and%20Organization (see the surfaces part in the image).